src/Snowflake/ExceptionHandler.php

```diff
@@ -23,8 +23,19 @@ public function __construct()
                 [1]
             ),
         ];
+
+        $this->messageTransformations[] =
```

Comment:
Bylo by fajn to ještě otestovat. To by imho mělo jít docela v pohodě pomocí toho `SELECT system$wait(70)`

Reply:
Jo to bych určitě mohl! :+1: 

Těch 60s timeoutu se bere v až v connection. Tady jsem udělal ruční nastavení tý SESSION proměnný a nechal to na 5s.